protobuf/syft_proto/messaging/v1/message.proto

```diff
@@ -40,3 +40,32 @@ message TensorCommandMessage {
     syft_proto.execution.v1.CommunicationAction communication = 2;
   }
 }
+
+message ForceObjectDeleteMessage {
+  syft_proto.types.syft.v1.Id object_id = 1;
+}
+
+message GetShapeMessage {
+  syft_proto.types.syft.v1.Id object_id = 1;
+}
+
+message ObjectRequestMessage {
+  syft_proto.types.syft.v1.Id object_id = 1;
+  string reason = 2;
+  // what type should user be? can't figure it out from docs/tests
+  string user = 3;
+}
+
+message PlanCommandMessage {
+  string command_name = 1;
+  // repeated ? args = 2;
```

Comment:
The type here can be `Args`, and we can expand the types in `Args` if we need to.